gomint-api/src/main/java/io/gomint/world/block/BlockSignNormal.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2020 Gomint team
+ *
+ * This code is licensed under the BSD license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package io.gomint.world.block;
+
+import io.gomint.world.block.data.SignDirection;
+
+/**
+ * @author geNAZt
+ * @version 1.0
+ * @stability 3
+ */
+public interface BlockSignNormal extends BlockSign {
```

Comment:
Please move BlockSign to BlockStandingSign and rename this class to BlockSign

Reply:
I'm not sure because i don't see the point of extending the BlockWallSign from BlockStandingSign 🤔

Reply:
Well there is no "special" sign. Thus there can't be a "normal" one when following to that logic. Also why should the WallSign be extended from the StandingSign? Those should both extend Sign.

Reply:
Currently Wall and Normal Sign are extending from BlockSign. I see the point of renaming Normal to Standing but not of renaming BlockSignNormal (BlockStandingSign) to BlockSign

Reply:
there should be three interfaces, BlockSign, BlockStandingSign and BlockWallSign. I don't see what there should be a BlockSignNormal

Reply:
Yes but you said: 
 `Please move BlockSign to BlockStandingSign and rename this class to BlockSign`

I understood it like
 * BlockSign (shared) -> BlockStandingSign
 * BlockSignNormal -> BlockSign (shared)

which makes actually no sense to me

Reply:
No,

- BlockWallSign -> BlockSign
- BlockStandingSign -> BlockSign

sorry for the confusion :D

Reply:
Ok i see! Done in b7e5e711001d2634a7592e70810918da12ea6f0a